CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2018-10-10 - Supported Release 2.1.0
+
+### Summary
+
+Feature release including support for Windows Server 2016 and Puppet 6
+
+#### Features
+
+- Add support for Windows Server 2016
+- Convert module for PDK ([MODULES-6459](https://tickets.puppetlabs.com/browse/MODULES-6459))
+- Add support for Puppet 6 ([MODULES-7832](https://tickets.puppetlabs.com/browse/MODULES-7832))
+
+#### Bug Fixes
+
+- Fixed issue with emitting change messages in Puppet 5 ([MODULES-5364](https://tickets.puppetlabs.com/browse/MODULES-5364))
```

Comment:
Hmm... as @ThoughtCrhyme has pointed out, this doesn't exist anywhere.

Reply:
Looks like it was for this commit... https://github.com/puppetlabs/puppetlabs-acl/commit/4f46a668cc0b825ce7192c608855b95c167e62ee. Which was really intended to build on https://github.com/puppetlabs/puppetlabs-acl/commit/028492823bcd4c37fffc8dfd7079ac2ff43c0133, which was marked with the same ticket number.

It perhaps should have been a `maint` commit (or separate ticket) rather than the release?

Anyhow - don't think we need it in the changelog.